### PR TITLE
Add books reducer

### PR DESCRIPTION
**Pull request by kwekubright**
### Description:
In this feature, I achieved the following:

- Added Redux Toolkit ✅ 
- Structured app using ducks pattern ✅ 
- Added book reducer ✅ 
- Added categories reducer ✅ 
- Configured redux store ✅ 
image

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export const CHECK_STATUS = 'CHECK_STATUS';
+
+export const initialState = {
+  status: '',
+};
```
Comment (thecodechaser) [inline src/redux/categories/categories.js]: - [ ] Please set the initial state to be an empty array as requested in the project requirements.

image

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export default function categoriesReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return {
+        ...state,
+        status: action.status,
+      };
```
Comment (thecodechaser) [inline src/redux/categories/categories.js]: - [ ] Please return a string "Under construction" For the check status action as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kwekubright👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @kwekubright,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**